tests/unit/test_download.py

```diff
@@ -127,7 +127,12 @@ def test_path_to_url_unix():
 
 @pytest.mark.skipif("sys.platform == 'win32'")
 def test_url_to_path_unix():
+    assert url_to_path('file:tmp') == 'tmp'
     assert url_to_path('file:///tmp/file') == '/tmp/file'
+    assert url_to_path('file:/path/to/file') == '/path/to/file'
+    assert url_to_path('file://localhost/tmp/file') == '/tmp/file'
```

Comment:
I think it would help readability if these were ordered `file:`, `file:/`, `file://`, `file:///`, etc.

I also think this and the other test is a good candidate for `@pytest.mark.parametrize()`.